beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/handler/ExpiredMetadataHandler.java

```diff
@@ -98,13 +98,14 @@ private boolean cleanup(CleanerClient client, HousekeepingMetadata housekeepingM
     return false;
   }
 
+
   private void cleanUpTable(CleanerClient client, HousekeepingMetadata housekeepingMetadata) {
     String databaseName = housekeepingMetadata.getDatabaseName();
     String tableName = housekeepingMetadata.getTableName();
     log.info("Cleaning up metadata for \"{}.{}\"", databaseName, tableName);
     if (metadataCleaner.tableExists(client, databaseName, tableName)) {
-      metadataCleaner.dropTable(housekeepingMetadata, client);
       pathCleaner.cleanupPath(housekeepingMetadata);
+      metadataCleaner.dropTable(housekeepingMetadata, client);
```

Comment:
Because it ended up dropping meta data before actually dropping the files, which was misleading when tracking the problem, because looking at hive meta store, the table wasn't listed anymore. so i thought this would help better reflect what the process is actually doing

Reply:
@abhimanyugupta07 not sure whether there are implications for the delete order here 🤔

Reply:
restored the deletion order